### ISSUE TITLE
`bordercolor` for `<A>`

### ISSUE THREAD

**Issue by pr-apes** [speedata/publisher#526]
@pgundlach,

would it be possible that `<A>` accepts the `bordercolor` attribute that works the same as `hyperlinkscolor` for `<Options>`.

Having different types of links (internal and external to the current PDF document), having a way to mark them as different is of great help to the reader (and even the writer).

Many thanks for your help.

**Comment by pgundlach** [comment]
This is a good idea. Probably only useful for Adobe Acrobat and the other viewers that display the border.

**Comment by pr-apes** [comment]
I tried to implement this myself, but I failed (I don't know how to create `a.bordercolor` in `commands.lua`).

Otherwise, I think it would only require to replace

``` lua
getBordercolor(options.hyperlinksbordercolor)
```

with

``` lua
getBordercolor(a.bordercolor or options.hyperlinksbordercolor)
```
Of course, it will work only in PDF viewers that display the annotation borders (and allow colored annotation borders).

**Comment by pgundlach** [comment]
I'll take a look, but this might take a (hopefully short) while.

**Comment by pgundlach** [comment]
Just a note for me: the `commands.xml` can be extended like this:

```xml
    <attribute en="bordercolor" type="text" optional="yes" since="4.15.7">
      <description xml:lang="en">
        <para>Set the color of the border (Adobe Acrobat ony).</para>
      </description>
      <description xml:lang="de">
        <para>Setze die Rahmenfarbe des links (nur Adobe Acrobat)</para>
      </description>
    </attribute>
```

(I need to take a closer look)

**Comment by pgundlach** [comment]
Implemented in version 4.15.7 - you can try it out if you want.

**Comment by pr-apes** [comment]
Many thanks for your new release, @pgundlach.

It works perfectly fine.
